### PR TITLE
fix: closedFn-undefined

### PR DESCRIPTION
**Pull request by Rishabh-Rathod**
## Description

Fixing the typo related to function name.

Fixes https://github.com/appsmithorg/appsmith-ee/issues/1172

Media

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Chore (housekeeping or task changes that don't impact user perception)
- This change requires a documentation update

- Manual
- Jest
- Cypress

### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/src/workers/Evaluation/evaluate.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const EvaluationScripts: Record<EvaluationScriptType, string> = {
   $$closedFn(${ScriptTemplate})
   `,
   [EvaluationScriptType.ASYNC_ANONYMOUS_FUNCTION]: `
-  async function callback (script) {
+  async function closedFn (script) {
```
Comment (arunvjn) [inline app/client/src/workers/Evaluation/evaluate.ts:49]: This should be $$closedFn right ?

Comment (Rishabh-Rathod) [reply app/client/src/workers/Evaluation/evaluate.ts]: yes, correct. I have pushed another commit to fix the typo.

**Review by arunvjn — APPROVED** [review]

**Comment by eco-monk** [comment]
Local test:
https://www.loom.com/share/26b93f2b7df7419796103cb78af4d0a6

**Pull request merged**